### PR TITLE
Desktop Header Component

### PR DESCRIPTION
**Pull request by savannahostrowski**
This PR contains a header component for desktop. This PR should be merged AFTER #8 as I will need to clean some things up once that one is merged for consistency's sake.

image

### CODE REVIEW

File: src/components/DesktopHeader.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      <div>
+        <AppBar position="static" className="appBar">
+          <div className="logo"/>
+          <h1 className="headerTitle" onClick={() => history.push('/')} style={{ cursor: 'pointer' }}>
+            Urban Carnivore Sightings
+          </h1>
+          <Button className={classes.moreIcon} onClick={this.toggleDrawer('right', true)}><Menu/></Button>
+          <SwipeableDrawer
+            disableBackdropTransition={!iOS}
+            disableDiscovery={iOS}
```
Comment (nachtm) [inline src/components/DesktopHeader.js]: What's going on here?

File: src/components/DesktopHeader.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  moreIcon: {
+    float: 'right',
+    display: 'inline',
+    color: 'white',
+    paddingBottom: 0,
+  },
+
+};
+
+const iOS = process.browser && /iPad|iPhone|iPod/.test(navigator.userAgent);
```
Comment (nachtm) [inline src/components/DesktopHeader.js]: Do iPhones or iPods ever get the Desktop view anyways?

File: src/components/DesktopHeader.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                </ListItem>
+                <ListItem className="drawerItem"><h3>Stub</h3>
+                </ListItem>
+                <ListItem className="drawerItem"><h3>Stub</h3>
+                </ListItem>
+              </List>
+            </div>
+          </SwipeableDrawer>
+          <div className="nav">
+            <div id="explore" className="categories" onClick={() => history.push('/map')}><h4>Explore</h4></div>
```
Comment (nachtm) [inline src/components/DesktopHeader.js]: 🔥

File: src/App.css [hunk 4]
```diff
@@ -2,6 +2,73 @@
...
+    left: 27%;
+}
+
+@media (min-width: 960px) {
+    .headerTitle {
+        left: 24%;
+    }
+}
+
+@media (min-width: 1440px) {
```
Comment (nachtm) [inline src/App.css]: This is really slick. Are 960/1440px pretty standard values for these kinds of breakpoints? Also 20/24/27% or did you come up with that on your own?

File: src/App.css [hunk 5]
```diff
@@ -2,6 +2,73 @@
...
+    .headerTitle {
+        left: 20%;
+    }
+}
+
+.logo {
```
Comment (nachtm) [inline src/App.css]: One mega-mega-mega nit is that I don't think this logo has a drop-shadow like the rest of the header does. IDK how tricky that is to make happen -- maybe something for the backlog?

File: src/components/DesktopHeader.js (hunk 2 above)
Comment (savannahostrowski) [reply src/components/DesktopHeader.js]: So this was a problem with Material UI and how iOS devices handle the Swipeable Drawer component. I copied this logic from OSM Quality Web but tracked down a GitHub issue that outlines some of the issues that were happening.

https://github.com/mui-org/material-ui/issues/12057

I'm going to remove this logic for now but when we test on iOS, let's check this out.

File: src/components/DesktopHeader.js (hunk 1 above)
Comment (savannahostrowski) [reply src/components/DesktopHeader.js]: Refer below!

File: src/App.css (hunk 4 above)
Comment (savannahostrowski) [reply src/App.css]: Mmm, I guess we should actually standardize these some more.

Typically we use 768px and 1280px etc. (https://medium.com/@uiuxlab/the-most-used-responsive-breakpoints-in-2017-of-mine-9588e9bd3a8a)

I'm going to rework these a bit. 

The percentages were defined by me to accommodate the breakpoints.

File: src/App.css (hunk 5 above)
Comment (savannahostrowski) [reply src/App.css]: Just added a v. small box shadow for now.

Comment (savannahostrowski) [reply src/App.css]: Also, we need a better asset for the logo - this is a screenshot 😆

Comment (savannahostrowski) [reply src/App.css]: We also might want to throw an onClick event on the logo to nav back to home eventually but that's tbd.

**Pull request merged**